physlr/physlr.py

```diff
@@ -2192,19 +2192,36 @@ def physlr_path_to_fasta(self):
         num_scaffolds = 0
         num_contigs = 0
         num_bases = 0
+
+        unused_seqs = {name[0:-1] for path in paths for name in path if name[-1] == "."}
+
+        gaps = ("N" * self.args.gap_size)
+
         for path in progress(paths):
-            # Remove unoriented sequences.
-            path = [name for name in path if name[-1] != "."]
             if not path:
                 continue
 
-            seq = "NNNNNNNNNN".join(Physlr.get_oriented_sequence(seqs, name) for name in path)
+            seq = gaps.join(Physlr.get_oriented_sequence(seqs, name)
+                            if name[-1] != "." else ("N" * len(seqs[name[0:-1]]))
+                            for name in path)
+
             if len(seq) < self.args.min_length:
                 continue
             num_scaffolds += 1
             print(f">{str(num_scaffolds).zfill(7)} LN:i:{len(seq)} xn:i:{len(path)}\n{seq}")
             num_contigs += len(path)
             num_bases += len(seq)
+
+        for name in seqs:
+            if name in unused_seqs:
+                seq = seqs[name]
+                if len(seq) < self.args.min_length:
+                    continue
```

Comment:
Do we print out all sequences -- even those that cannot be mapped to the backbone?

Reply:
This was to address the drop in genome%

Reply:
Yes that's the behavior we want - I just wanted to double check that we were in fact printing everything out (unoriented and unmapped)

Reply:
Oh I see what you mean

Reply:
The funny thing is I implemented it in the current way (printing unmapped) originally, but I changed to what you saw because at that time I couldn't remember why I implemented that way.